core/src/main/java/com/android/volley/toolbox/NetworkUtility.java

```diff
@@ -50,7 +50,7 @@ private NetworkUtility() {}
     /** Logs requests that took over SLOW_REQUEST_THRESHOLD_MS to complete. */
     static void logSlowRequests(
             long requestLifetime, Request<?> request, byte[] responseContents, int statusCode) {
-        if (VolleyLog.DEBUG || requestLifetime > SLOW_REQUEST_THRESHOLD_MS) {
+        if (VolleyLog.DEBUG && requestLifetime > SLOW_REQUEST_THRESHOLD_MS) {
```

Comment:
Let's remove the "requestLifetime > SLOW_REQUEST_THRESHOLD_MS" check altogether and just log this when VolleyLog.DEBUG is true.

I think we should retain the behavior that when DEBUG is true, we log all requests, not just slow ones. So we can remove all of the logic around slow requests, including the SLOW_REQUEST_THRESHOLD_MS constant.